test/kind/Makefile

```diff
@@ -18,24 +20,43 @@ dev:
 	docker image tag calicovpp/multinet-monitor:$(TAG) localhost:5000/calicovpp/multinet-monitor:$(TAG)
 	docker push localhost:5000/calicovpp/multinet-monitor:$(TAG)
 
+# this is for a cluster using a mirror docker.io -> localhost:5000 to avoid pull rate limit
+.PHONY: load-mirror
+load-mirror:
+	docker pull docker.io/calico/node:$(KIND_CALICO_VERSION)
```

Comment:
Shouldn't we be using a list of image names (as we are using them in a few place, just to make sure we don't miss one in one of the directives). We're using a similar pattern for e.g. `make dev.k3s` in the top level Makefile